packages/eds-core-react/src/components/Radio/Radio.stories.tsx

```diff
@@ -116,6 +124,28 @@ export const AlternativeToLabel: Story<RadioProps> = () => (
 )
 AlternativeToLabel.storyName = 'Alternative to label'
 
+export const CustomLabel: Story<RadioProps> = () => {
+  return (
+    <div style={{ display: 'contents' }}>
```

Comment:
I don't think this is the correct usage of display: contents and the end result is a bit weird witht the distance between label and radio. How about display flex , align-items center.

Reply:
Pure provocation with `contents`, but yes you right.

Reply:
done